kpm/lib/kpm/diagnostic_file.rb

```diff
@@ -149,9 +149,9 @@ def get_log_files(log_dir)
       # Helpers
 
       def get_system_catalina_base(export_data)
-
         system_json = JSON.parse(export_data)
         @catalina_base = system_json['java_system_information']['catalina.base']['value']
+        @catalina_base = @catalina_base + File::Separator + 'logs'
```

Comment:
I would instead do `log_base = log_dir || (@catalina_base + File::Separator + 'logs')` above, otherwise it's a bit confusing (`@catalina_base` would point to the logs sub directory, instead of `$CATALINA_BASE`).